Author: Robert James Clay <jame@rocasa.us>
Description: Correct minor spelling errors in LedgerSMB/DBObject/Payment.pm.
Origin: vendor
Last-Update: 2016-03-31
--- a/LedgerSMB/DBObject/Payment.pm
+++ b/LedgerSMB/DBObject/Payment.pm
@@ -281,7 +281,7 @@
 =item $payment->get_open_invoice()
 
 This function is an especific case of get_open_invoices(), because get_open_invoice() 
-can search for a specific invoice, wich can be searched by the $payment->{invnumber} 
+can search for a specific invoice, which can be searched by the $payment->{invnumber} 
 variable 
 
 =back
@@ -421,7 +421,7 @@
 This method gets a list of the open currencies inside the database, it requires that  
 $self->{account_class} (must be 1 or 2)  exist to work. 
 
-WARNING THIS IS NOT BEEING USED BY THE SINGLE PAYMENT SYSTEM.... 
+WARNING THIS IS NOT BEING USED BY THE SINGLE PAYMENT SYSTEM.... 
 
 =back
 
--- a/LedgerSMB/DBObject.pm
+++ b/LedgerSMB/DBObject.pm
@@ -36,7 +36,7 @@
 it will be up to the application to handle any exceptions.
 
 =item __validate__ is called on every new() invocation.  It is blank in this 
-module but can be overridden in decendant modules.
+module but can be overridden in descendant modules.
 
 =item _db_array_scalars(@elements) creates a db array from scalars.
 
--- a/LedgerSMB/File.pm
+++ b/LedgerSMB/File.pm
@@ -279,7 +279,7 @@
 
 =item get
 
-Retrives a file.  ID and file_class properties must be set.
+Retrieves a file.  ID and file_class properties must be set.
 
 =cut
 
